.travis.yml

```diff
@@ -5,7 +5,9 @@ php:
   - 5.4
   - 5.5
   - 5.6
-  - 7
+  - 7.0
+  - 7.1
+  - nightly
```

Comment:
We could move nightly to allowed to fail so we know if it works on PHP 8 and PHP 9 before it comes out :smile: